db/seeds.rb

```diff
@@ -1,7 +1,16 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+titles = [
+  'New Vaccine Conspiracy Theories Are Going Viral in Arabic',
+  '‘It is a trap!’: Inside the QAnon attack that never happened',
+  'New Year 2020: 10 Moments This Decade That Made Us Say, "Wait, What?"',
+"Amateur Rocket-Maker Finally Launches Himself Off Earth - Now To Prove It's Flat"
+]
+teasers = [
+  'Facebook has been criticized for failing to curb misinformation in English. But little attention has been paid to the scale of the problem in Arabic.',
+'Why fears of violence on March 4, the mythical day Trump was supposed to be inaugurated to a second term, proved unfounded.',
+'Happy New Year 2020: This list of ten moments includes not just personalities in India and across the world, but also certain events which have had a number of consequences for those involved.',
+'Mike Hughes, a California man who is most known for his belief that the Earth is shaped like a Frisbee, finally blasted off into the sky in a steam-powered rocket he had built himself.'
+]
+
+for i in 0...titles.count
```

Comment:
Wow! Why not keep it simple and use 

```ruby
titles.each do |title|
  # whatever you need to  do with each 'title'
end

Reply:
We would need the index to access teaser, but could use `titles.each_with_index` for that, indeed!

Reply:
Also we suck at `for` loops, so good practice 😄

Reply:
nah... you would have teaser in the `title['teaser']` won't you? 

sucking at for loops is not a bad thing... if you can avoid that syntax. `.each` IS a for loop

Reply:
Forget it. I see what you are doing now.... Okay. But mark my words, this is NOT a low-hanging fruit.